_includes/footer.html

```diff
@@ -1,18 +1,22 @@
 <!-- This will be our footer -->
-<footer class="footer mdl-mega-footer">
-  <div class="mdl-mega-footer__bottom-section">
-    <div class="mdl-mega-footer__drop-down-section">
-      <div class="mdl-mega-footer__heading" >Hack For LA</div>
-      <p class="footer_slogan">100 Automations is one of the many projects at Hack for LA, Code for America’s Los
-        Angeles chapter.
+<footer class="footer mdl-mini-footer">
```

Comment:
This references a .footer class that doesn't seem to exist anywhere.  (Unless I'm not seeing it.)  If it doesn't exist, I'd recommend removing the reference to it.